week0004/pertsodian/java/src/event/management/domain/Guest.java

```diff
@@ -0,0 +1,92 @@
+package event.management.domain;
+
+public class Guest implements Comparable<Guest> {
+
+	private static final String REGISTRATION_ENTRY_DELIMETER = ",";
+
+	private String firstName;
+	private String lastName;
+	private String emailAddress;
+
+	public Guest(String firstName, String lastName, String emailAddress) {
+		this.firstName = firstName;
+		this.lastName = lastName;
+		this.emailAddress = emailAddress;
+	}
+
+	public static Guest fromRegistrationEntry(String registrationEntry) {
```

Comment:
Not advised to have such static method inside the class. You can create a separate function or a utility function/class. Now the class `Guest` bears responsibility of parsing registration entry => too much responsibility.